src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,183 @@
-const Countdown = () => null;
+import { useEffect, useRef, useState } from 'react';
+import { STATUS } from '../../utils/constants';
+import type { StatusType } from '../../utils/constants';
+import { DisplayForText, DisplayForTime, convertToSeconds } from '../../utils/helpers';
+import { Button, Buttons, Input, Inputs, SupportText, TimeDisplay, Timer, TimerContainer, TimerTitle } from '../../utils/styles';
+
+const Countdown = () => {
+    const [timeMinInput, setTimeMinInput] = useState('');
+    const [timeSecInput, setTimeSecInput] = useState('');
+
+    const [status, setStatus] = useState<StatusType>(STATUS.INITIAL);
+    const [secondsRemaining, setSecondsRemaining] = useState(0);
+
+    const secondsOnTimer = secondsRemaining % 60;
+    const minutesRemaining = (secondsRemaining - secondsOnTimer) / 60;
+    const minutesOnTimer = minutesRemaining % 60;
+    const hoursOnTimer = (minutesRemaining - minutesOnTimer) / 60;
+
+    const totalSeconds = convertToSeconds(timeMinInput, timeSecInput);
+
+    const startStopCountdown = () => {
```

Comment:
You have some opportunities to DRY up your code. Wherever you're writing the same code twice, think about breaking it out into a helper function. Not only does that help with code readability, but if you have duped code, it's really easy to update the logic in one place and forget to update it in the other.

Ex. 
```
 if (Number.isNaN(totalSeconds) || (timeMinInput === '' && timeSecInput === '') || totalSeconds <= 0) {
            alert('Please enter a valid time.');
        }
```
We see this in a few of the timers. 

Maybe this guard could be refactored to a util function called `warnInvalidTime(totalSeconds, timeMinInput, timeSecInput)`. Or maybe you can add all your guards to a single shared function across all four components.